### PR TITLE
android: added logs to callback functions

### PR DESCRIPTION
**Pull request by pratikvelani**
- Implements all the callback functions and adds logs to it.

### CODE REVIEW

**Comment by pratikvelani** [comment]
@francesca64 I have implemented the callback functions in Kotlin and added logs to it, you can use this for Android connlib updates.

File: kotlin/android/app/src/main/java/dev/firezone/android/features/session/backend/SessionCallbackImpl.kt [hunk 1]
```diff
@@ -5,22 +5,44 @@ import dev.firezone.connlib.SessionCallback
...
+        // TODO: Call into client app to update resources list and routing table
+        Log.d(TAG, "onUpdateResources: $resources")
+    }
 
-        return true
+    override fun onSetInterfaceConfig(
+        tunnelAddressIPv4: String,
+        tunnelAddressIPv6: String,
+        dnsAddress: String,
+        dnsFallbackStrategy: String
```
Comment (jamilbk) [inline kotlin/android/app/src/main/java/dev/firezone/android/features/session/backend/SessionCallbackImpl.kt:17]: @roop I believe we need this also in the Apple client, correct?

**Review by jamilbk — COMMENTED** [review]
Thanks for adding these @pratikvelani

**Comment by jamilbk** [comment]
Just needs approval from @conectado and @francesca64

**Review by francesca64 — APPROVED** [review]

**Review by conectado — APPROVED** [review]

**Pull request merged**